rpcs3/Emu/Cell/lv2/sys_time.cpp

```diff
@@ -102,6 +102,23 @@ static int clock_gettime(int clk_id, struct timespec* tp)
 
 #endif
 
+#ifndef _WIN32
```

Comment:
Why ever scale wall clock in the first place? Wall clock is what sys_time_get_current_time returns, or CLOCK_REALTIME.

Reply:
For consistency, usleep tests for example rely on sys_time_get_current_time.

Reply:
usleep tests can survive without these changes. That's not persuasive to scale wall clock at all. It was supposed to be synchronized with accurate host clock periodically.